boot/seal.go

```diff
@@ -141,20 +139,43 @@ func fallbackKeySealRequests(key, saveKey secboot.EncryptionKey) []secboot.SealK
 	}
 }
 
+type fdeSetupHookResult struct {
+	EncryptedKey []byte `json:"encrypted-key"`
+	Handle       []byte `json:"handle"`
```

Comment:
How is handle used later on?

Reply:
it's meant to be passed back when we unseal the key

Reply:
afaict from the intention of secboot code, here we should probably use *json.RawMessage for Handle and not []byte, of course the hooks can send a base64 string and that's fine